modules/installing-ocp-agent.adoc

```diff
@@ -0,0 +1,167 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent_{context}"]
+= Installing {product-title} with the Agent-based Installer
+
+The following procedure deploys a single-node {product-title} and you can use this procedure as a basis and modify according to your requirements.
+
+.Procedure
+
+. Navigate to the link:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-{product-version}/[OpenShift mirror site] and download the latest version of the tarball that matches your operating system.
+
+. Extract the binary.
++
+[source,terminal]
+----
+$ tar xvf <file> -C ~/.local/bin
+----
++
+The file contains the Agent-based Installer.
+
+. Install `nmstate` dependency.
++
+[source,terminal]
+----
+$ sudo dnf install /usr/bin/nmstatectl -y
+----
+
+. Place the `openshift-install` binary in a directory that is on your PATH.
+
+. Create a directory to store the cluster manifests.
++
+[source,terminal]
+----
+$ mkdir ~/<directory_name>
+----
+
```

Comment:
Add a `+` before [NOTE] to align this with Step 5. It is breaking the numbering order.